instances/k8sworker/scripts/setup.template.sh

```diff
@@ -4,13 +4,10 @@ EXTERNAL_IP=$(curl -s -m 10 http://whatismyip.akamai.com/)
 NAMESPACE=$(echo -n "${domain_name}" | sed "s/\.oraclevcn\.com//g")
 FQDN_HOSTNAME=$(getent hosts $(ip route get 1 | awk '{print $NF;exit}') | awk '{print $2}')
 
-# make sure ubuntu owns home dir
-chown ubuntu:ubuntu /home/ubuntu
-
-# pull instance metadata
+# Pull instance metadata
```

Comment:
make sure proper ownership of opc home dir

Reply:
It's fine.

```
$ ls -all /home/
total 8
drwxr-xr-x.  3 root root   17 Oct 13 20:41 .
dr-xr-xr-x. 17 root root 4096 Sep 26 18:34 ..
drwx------.  5 opc  opc  4096 Oct 13 20:50 opc
```